app/wallets/AbstractWallet.ts

```diff
@@ -0,0 +1,39 @@
+import { Transaction } from "stellar-sdk";
+import ILocalStorage from "../interfaces/storage";
+import { FC } from "react";
+import IWallet from "../interfaces/wallet";
+import NotImplementedError from "./errors/NotImplementedMethod";
+
+export default abstract class AsbtractWallet implements IWallet {
```

Comment:
There's a typo here.

Reply:
I have corrected the typo by changing AsbtractWallet to AbstractWallet.